tests/IntegrationTests/TableView.fs

```diff
@@ -0,0 +1,57 @@
+module Pulsar.Client.IntegrationTests.TableView
+
+open System
+open System.Linq
+open Expecto
+open Expecto.Flip
+open Pulsar.Client.Api
+
+open Serilog
+open Pulsar.Client.IntegrationTests.Common
+open FSharp.Control
+
+[<Tests>]
+let tests =
+    testList "TableView" [
+        testAsync "TableView with non-partition-topic works fine" {
+            Log.Debug("Started testTableView")
+            let client = getClient()
+            let numberOfMessages = 10
+            let producerName = "tableViewProducer"
+            let topicName = "public/default/" + Guid.NewGuid().ToString("N")
+
+            let! producer =
+                client.NewProducer()
+                    .Topic(topicName)
+                    .ProducerName(producerName)
+                    .CreateAsync()  |> Async.AwaitTask
+
+            do! produceMessagesWithProps producer numberOfMessages producerName  |> Async.AwaitTask
```

Comment:
Why does this work? Usually producing to topic without consumer results in just discarding messages. 
Anyway, another idea is that it will be more useful to produce messages with specific key+value and check them below. For example key1+value1, key1+value2, key2+value3. And them check both keys below

Reply:
We only push the message with the key to `TableView`, the message without the key will be discarded.

Reply:
Yes, I meant that messages are produced to the topic before consumer (reader -> table view) is created. In all other tests message are only produced after consumer creation.